WooCommerce/Classes/ViewRelated/Products/Edit Product/Linked Products List Selector/LinkedProductsListSelectorViewController.swift

```diff
@@ -0,0 +1,206 @@
+import UIKit
```

Comment:
It's showed as a new file, but as you can see, there are just a small bunch of differences from the previous version. Github shows it as a new file just because it was moved under a new folder 🤷‍♂️